src/execution_plan/execution_plan.c

```diff
@@ -230,6 +230,20 @@ OpBase* ExecutionPlan_LocateOp(OpBase *root, OPType type) {
     return NULL;
 }
 
+void ExecutionPlan_Taps(OpBase *root, OpBase ***taps) {
+    if(root == NULL) return;
+    int t = OPType_ALL_NODE_SCAN |
+            OPType_NODE_BY_LABEL_SCAN |
+            OPType_INDEX_SCAN |
+            OPType_NODE_BY_ID_SEEK;
+
+    if(root->type & t) *taps = array_append(*taps, root);
+
+    for(int i = 0; i < root->childCount; i++) {
+        ExecutionPlan_Taps(root->children[i], taps);
```

Comment:
Once we have more execution plan options, we should make a more efficient iteration over the execution plan. This is fine for now, though!